v3/src/components/mosaic-tile-row.tsx

```diff
@@ -89,14 +92,14 @@ export const MosaicNodeComponent = observer(({ node, direction, pctExtent, ...ot
  */
 interface IMosaicTileProps extends IExtentProps {
   tile: ITileModel
-  row: IMosaicTileRow
+  content?: IDocumentContentModel
 }
-export const MosaicTileComponent = observer(({ row, tile, direction, pctExtent }: IMosaicTileProps) => {
+export const MosaicTileComponent = observer(({ content, tile, direction, pctExtent }: IMosaicTileProps) => {
   const style = styleFromExtent({ direction, pctExtent })
   const tileType = tile.content.type
   const info = getTileComponentInfo(tileType)
   const handleCloseTile = (tileId: string) => {
-    row?.removeTile(tileId)
+    content?.deleteTile(tileId)
```

Comment:
I think for now you probably have to do both:
```typescript
    content?.deleteTile(tileId)
    row?.removeTile(tileId)
```
With some additional work we could make it the case that the row models respond to content changes, but that's not in place right now, so manually keeping the two in sync will presumably be required for the time being.